internal/metastore/db/dao/collection_alias.go

```diff
@@ -0,0 +1,105 @@
+package dao
+
+import (
+	"time"
+
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/metastore/db/dbmodel"
+	"github.com/milvus-io/milvus/internal/util/typeutil"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+)
+
+type collAliasDb struct {
+	db *gorm.DB
+}
+
+func (s *collAliasDb) Insert(in []*dbmodel.CollectionAlias) error {
+	err := s.db.CreateInBatches(in, 100).Error
+	if err != nil {
+		log.Error("insert collection alias failed", zap.Error(err))
+		return err
+	}
+
+	return nil
+}
+
+func (s *collAliasDb) Update(tenantID string, collID typeutil.UniqueID, alias string, ts typeutil.Timestamp) error {
```

Comment:
`ts` here is a little strange. It means the ts of event of `AlterAlias` or the `ts` of collection?

Reply:
alias also needs time-travel

Reply:
OK, it's the ts of alias. Got it.